packages/tabs/src/Tab.tsx

```diff
@@ -2,7 +2,7 @@ import { Tab as ReachTab } from "@reach/tabs"
 import React, { ReactNode, useState, SyntheticEvent } from "react"
 import classnames from "classnames"
 import { Badge } from "@kaizen/draft-badge"
-import styles from "./styles.scss"
+import styles from "./Tabs.module.scss"
```

Comment:
`Tabs.tsx` doesn't actually pull in any styles itself, but the stylesheet infers that the styles are for that component.

See if it's possible to break out the styles into files relating to the components using them.